aem-intellij-core/src/test/kotlin/com/aemtools/codeinsight/osgiservice/DsOSGiPropertyLineMarkerTest.kt

```diff
@@ -0,0 +1,427 @@
+package com.aemtools.codeinsight.osgiservice
+
+import com.aemtools.codeinsight.osgiservice.markerinfo.FelixOSGiPropertyMarkerInfo
+import com.aemtools.common.constant.const
+import com.aemtools.test.base.BaseLightTest
+import com.aemtools.test.fixture.JavaMixin
+import com.aemtools.test.fixture.OSGiConfigFixtureMixin
+import com.aemtools.test.fixture.OSGiDsAnnotationsMixin
+import com.intellij.codeInsight.daemon.LineMarkerInfo
+import org.assertj.core.api.Assertions
+
+/**
+ * Test for [DsOSGiPropertyLineMarker].
+ *
+ * @author Kostiantyn Diachenko
+ */
+class DsOSGiPropertyLineMarkerTest : BaseLightTest(),
+    OSGiConfigFixtureMixin,
+    OSGiDsAnnotationsMixin,
+    JavaMixin {
+
+  fun `test marker info DS OSGi Config with OCD in separate class`() = fileCase {
+    addComponentAnnotation()
+    addDesignateAnnotation()
+    addObjectClassDefinitionAnnotation()
+    addAttributeDefinitionAnnotation()
+
+    javaLangString()
+
+    addClass("Config.java", """
+      package com.test;
+
+      import ${const.java.DS_ATTRIBUTE_DEFINITION_ANNOTATION};
+      import ${const.java.DS_OBJECT_CLASS_DEFINITION_ANNOTATION};
```

Comment:
While that's a common practice across programming languages and frameworks to reuse production constants in the test code, that's bad practice.
Among of the features, your tests should be providing you is the safety net against regression that might be caused by the changes in you _production_ code, with the above setup, if you would accidentally put a typo in one of those constants the test will still be green. Not good

Reply:
Fixed, thank you